### PR TITLE
Handle status failure for sign tx (#786)

### PR DESCRIPTION
**Pull request by marcinbodnar**

### CODE REVIEW

**Comment by marcinbodnar** [comment]
@vgrichina It's not yet fixed for 2fa, but sign with 2fa doesn't work for now.

**Comment by kcole16** [comment]
@marcinbodnar Looks good. Makes sense that 2FA doesn't work yet (until we update near-api-js) I believe cc @mattlockyer

File: src/utils/wallet.js [hunk 1]
```diff
@@ -708,7 +708,10 @@ class Wallet {
         }
         for (let { receiverId, nonce, blockHash, actions } of transactions) {
             const [, signedTransaction] = await nearApiJs.transactions.signTransaction(receiverId, nonce, actions, blockHash, this.connection.signer, accountId, NETWORK_ID)
-            await this.connection.provider.sendTransaction(signedTransaction)
+            let { status } = await this.connection.provider.sendTransaction(signedTransaction)
+            if (status.Failure !== undefined) {
+                throw new Error('Transaction failure')
```
Comment (vgrichina) [inline src/utils/wallet.js:713]: let's include transaction hash and receiver in the message (to allow easy lookup for what went wrong)

Comment (marcinbodnar) [reply src/utils/wallet.js]: done

**Pull request merged**